synapse/storage/databases/main/room.py

```diff
@@ -1630,6 +1636,8 @@ def _get_event_reports_paginate_txn(
             sql = """
                 SELECT COUNT(*) as total_event_reports
                 FROM event_reports AS er
+                LEFT JOIN events
+                    ON events.event_id = er.event_id
```

Comment:
Short-hand if we want. It looks like we're just matching the query below though :+1: 

```suggestion
                LEFT JOIN events USING(event_id)
```